pkgs/development/interpreters/spidermonkey/common.nix

```diff
@@ -160,6 +174,6 @@ stdenv.mkDerivation (finalAttrs: rec {
     homepage = "https://spidermonkey.dev/";
     license = licenses.mpl20;
     maintainers = with maintainers; [ abbradar lostnet catap ];
-    platforms = platforms.linux;
+    platforms = platforms.unix;
```

Comment:
Does this work on Darwin for all versions or just 102? The conditional expressions above seem to hint at the latter.

Reply:
@jtojnar yes, it does. See included tests which was successfully executed by ofborg